next.clayui.com/content/docs/css/components/stickers.md

```diff
@@ -0,0 +1,381 @@
+---
+title: "Stickers"
+description: "Stickers are a visual way to quickly identify content in a different way than badges and labels."
+---
+
+<div class="nav-toc">
+
+- [Colors](#colors)
+- [Position](#position)
+- [Sizes](#sizes)
+- [Variations](#variations)
+    - [Overlay](#overlay)
+	- [Outside](#outside)
+	- [User Icon](#user-icon)
```

Comment:
Should these two be indented at the same level as overlay?

Reply:
That's weird, on my editor everything looks fine: 
<img width="204" alt="Screen Shot 2019-05-30 at 17 01 17" src="https://user-images.githubusercontent.com/7663513/58660790-a39f3400-82fc-11e9-958d-b81677bc3967.png">

Reply:
Do you have mixed tabs and spaces by chance? Sometimes it helps to toggle on whitespace in your editor too `"editor.renderWhitespace": "all",`

Reply:
Prettier can format Markdown, although I am not sure whether we have any "weird" constructs in our Markdown that might cause issues.

Reply:
Prettier considers html markup? if you add whitespace between the second-level markups and more, this breaks the gatsby.

Reply:
Prettier works with normal markdown but I'm not positive about gatsby's version which I think is [MDX](https://www.gatsbyjs.org/docs/mdx/#whats-mdx)?

Reply:
I'm already using `"editor.renderWhitespace": "all"`.

Reply:
Fixed just removing white spaces then tabbing again.

Reply:
yeah, we used a mix of the two, MDX and MD, we had to do that because many MDX were leaving the build and watch slow, we have cases that do not have to be interpreted by an MDX.

Reply:
I see only three files with an ".mdx" extension, so can we apply Prettier to the Markdown ones? On the other hand, if we have ".md" files which are actually MDX, then maybe we should give them that extension so as to be explicit.

Reply:
hey @wincent  I would not do that 😅,for two reasons the `MDX` files are interpreted by the `gatsby-mdx` plugin which in the end treats them as React components this requires more processing and when we were using `clayui.com` using just `.mdx` we had almost 90 files or more, generated a huge slowness. Processing files with `.md` with the `gatsby-transformer-remark` plugin is much lighter, the idea I had is that `.mdx` files are only for cases where you need to render a React component in our case. pages of the React Components and we can stay with the `.md` files for the CSS pages that are simpler.

Maybe just format the `.md` files?

Reply:
> Maybe just format the .md files?

That's exactly what I've been suggesting. But if I understand you correctly, you're saying that many of the ".md" files are actually MDX files?

Reply:
Oh no, sorry for generating a confusion here 😅. I had understood that it was for us to change the files to `.mdx`, which was to say in the end that both are different and that the `.mdx` files are heavier and that they should only be used if need be.

Reply:
Ok, so to summarize:

- Prettier can format the `.md` for us, which will help us avoid these little whitespace issues coming in.
- It cannot format `.mdx`, but that's fine because there are only three of them in the repo.
- We are in no hurry to switch existing `.md` to `.mdx` because that would only slow things down.

Reply:
Yes, great. Just an addendum, maybe we can increase the number of `.mdx` files because we will add more documents from the components.